index.css

```diff
@@ -212,4 +212,32 @@ footer {
 
 footer .license {
 	float: right;
+}
+
+footer br{
+	display: none;
+}
+
+h2{
+	text-align: center;
+}
+
+@media only screen and (max-width: 746px) {
+	section{
+		padding-left: 
```

Comment:
This is not syntactically valid.

Reply:
I express regret for neglecting the half written line. This line is of no use and was meant to be removed.